src/Element/WebformPrivacyStatement.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Drupal\tide_webform\Element;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Element\Checkbox;
+
+/**
+ * Provides a webform Privacy Statement element.
+ *
+ * @FormElement("webform_privacy_statement")
+ */
+class WebformPrivacyStatement extends Checkbox {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getInfo() {
+    return [
+      '#return_value' => TRUE,
+      '#privacy_statement_heading' => '',
+      '#privacy_statement_content' => '',
+    ] + parent::getInfo();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function preRenderCheckbox($element) {
+    $element = parent::preRenderCheckbox($element);
+    $id = 'webform-privacy-statement-' . implode('_', $element['#parents']);
+
+    if (empty($element['#title'])) {
+      $element['#title'] = (string) t('I have read and understand the privacy statement.');
```

Comment:
Please check grammar here... I think it should be `I have read and understood...` but it can stand as is.